infra/attributes/default.rb

```diff
@@ -0,0 +1,3 @@
+default['postgresql']['initial_root_password'] = 'password'
+default['postgresql']['password']['postgres'] = 'password'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.